### PR TITLE
Update FlyteCTL Contribution Guide

### PR DESCRIPTION
**Pull request by samhita-alla**
Signed-off-by: Samhita Alla <aallasamhita@gmail.com>

Updated README and contribution guide

## Read then delete

- Make sure to use a concise title for the pull-request.
- Use #patch, #minor #majora or #none in the pull-request title to bump the corresponding version. Otherwise, the patch version
  will be bumped. [More details](https://github.com/marketplace/actions/github-tag-bump)

# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Plugin

## Are all requirements met?

- [ ] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
_How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Install the requirements by running ``pip install -r doc-requirements.txt`` in the root folder
+2. Make modifications in the `docs/source/gen <https://github.com/flyteorg/flytectl/tree/master/docs/source/gen>`__ folder
+3. Run ``make gendocs`` from within the `docs <https://github.com/flyteorg/flytectl/tree/master/docs>`__ folder
+4. Open html files produced by Sphinx in your browser to verify if the changes look as expected (html files can be found in the ``docs/build/html`` folder)
+
+💻 Contribute Code
+==================
+
+1. Set up a local cluster by running ``flytectl sandbox start --source=$(pwd)`` in the root directory
+2. xxx
```
Comment (samhita-alla) [inline docs/source/contribute.rst:24]: @pmahindrakar-oss @evalsocket can someone please complete this? I have deleted the CONTRIBUTING.md file as I'm adding this directly to the documentation.

Comment (samhita-alla) [reply docs/source/contribute.rst:24]: I'll add the troubleshooting techniques mentioned in the markdown to the troubleshooting guide later.

Comment (pmahindrakar-oss) [reply docs/source/contribute.rst:24]: Next steps would be to compile the latest changes for flytectl from the repo

And run bin/flytectl get projects as an example to see if things are working and later the contributor can test the specific command



@samhita-alla you can run through these steps and update those here.

**Review by pmahindrakar-oss — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -10,42 +25,47 @@
...
-Generating docs locally can be accomplished by running make gendocs from within the docs folder
+* Install FlyteCTL with bash or shell script
 
+    * Bash
+        ```bash
+        $ brew install flyteorg/homebrew-tap/flytectl
 
-## Installation
+        # Upgrade flytectl 
+        $ brew upgrade flytectl
```
Comment (evalsocket) [inline README.md:39]: Can we remove the brew upgrade and add a common section for an upgrade that will use `flytectl upgrade`

**Review by evalsocket — APPROVED** [review]

**Review by evalsocket — APPROVED** [review]

**Pull request merged**